compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -4408,9 +4388,7 @@ static void unsignedIntCompareNarrower(TR::Node * node, TR::Simplifier * s, TR::
          }
       else if (firstOp == TR::b2i && firstChild->getReferenceCount() == 1)
          {
-         if (secondOp == TR::b2i ||
-             (secondOp == TR::iuconst               &&
```

Comment:
shouldn't this be an iconst with a getUnsignedInt to test against SCHAR_MAX?

Reply:
Fixed.